it_hardware/demo/website_menu.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="website_menu_12" model="website.menu">
+        <field name="name">Bulk Order</field>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="url">/bulk-order</field>
```

Comment:
As mentioned in the pad, this page doesn't seem viable with the hardcoded list of products.

Reply:
Then you can remove all related (ir.ui.view, website.page, ...)

Reply:
up